data/campaigns/Descent_Into_Darkness/scenarios/07b_A_Small_Favor2.cfg

```diff
@@ -258,8 +258,9 @@
                 [/not]
             [/filter_location]
             [command]
+                {NAMED_NOTRAIT_UNIT 2 "$guardtypetable[$($loc.n % $guardtypetable.length)].type"
                 # po: Unit name
-                {NAMED_NOTRAIT_UNIT 2 "$guardtypetable[$($loc.n % $guardtypetable.length)].type" $loc.x $loc.y () (_ "Guard")
+                $loc.x $loc.y () (_ "Guard")
```

Comment:
Would the WML be clearer if the whole thing could be put on one line (including the line that's currently split), and have a single comment that's suitable for both genders?

Maybe `# po: A role-based name, will be used for soldiers of the spearman, fencer and heavy infantry lines`.

Allowing two translatable strings on the same line might be out-of-scope for the current PR.

Reply:
I was thinking we could have `# po: ...` apply a comment to the first translatable string on the next line and `# po-all: ...` apply it to all translatable strings on the next line, but I'm not sure if that's a good design.